apis/v1beta1/validation/httproute.go

```diff
@@ -41,6 +42,9 @@ var (
 
 	// All valid path characters per RFC-3986
 	validPathCharacters = "^(?:[A-Za-z0-9\\/\\-._~!$&'()*+,;=:@]|[%][0-9a-fA-F]{2})+$"
+
+	// Minimum value of timeout fields
+	timeoutMinValue = time.Millisecond * 1
```

Comment:
Why do we need this? "ms" is the smallest unit we support and we don't support fractional values. We also support "0" which makes this value particularly confusing.

Reply:
This is left over from before we changed to the restricted syntax where 1ms is the minimum value you can represent.

Seems like we can completely remove the min value checking from the validator now that anything less is not representable in the syntax. Only thing to check now is that BackendRequest <= Request timout and that 0 is allowed.

Reply:
addressed in e56b07bba434dcad3253d6ff03a07083f1aa198d